src/Dumplie/Domain/SharedKernel/Money/Currencies.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\SharedKernel\Money;
+
+final class Currencies
```

Comment:
To be honest I didn't check it, current solution should be enough at least for now, if we found any case where simple array with currency codes is not enough then we gonna need to think about changing it to something else.